app/src/main/java/org/mozilla/rocket/content/travel/ui/TravelMapper.kt

```diff
@@ -108,10 +107,10 @@ object TravelMapper {
                 ig.linkUrl
             )
 
-    fun toExploreWikiUiModel(wiki: Wiki): WikiUiModel =
+    fun toExploreWikiUiModel(wiki: Wiki, source: String): WikiUiModel =
             WikiUiModel(
                 wiki.imageUrl,
-                SOURCE_WIKI,
```

Comment:
We do need a property to keep non-translated source name. Please add a new property in `WikiUiModel` to show translated content